zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/cacheAPI/SendNotificationCacheApi.java

```diff
@@ -0,0 +1,43 @@
+package com.github.onsdigital.zebedee.model.publishing.cacheAPI;
```

Comment:
You need to go through all your changes and ensure that you use the correct name (Legacy Cache API, not just Cache API). There are multiple places in Zebedee and Babbage where caching is being handled, so it's important to write accurate names, otherwise it will get confusing for the maintainers of these projects.